pkg/finder/finder.go

```diff
@@ -114,6 +119,14 @@ func NewFinder(args Args, pciResMapConf map[string]string) (CRIFinder, error) {
 	}
 	var err error
 
+ 	// At this stage, we only support containerd and cri-o
+	if args.ContainerRuntime == "containerd" {
```

Comment:
@fromanirh Let me know if this check is reasonable to do or if you think we should do it in another way.

Reply:
it is totally reasonnable, let's just use package-level constants and let's add explicit validation instead of silently handling errors. E.g. if the user provides `--container-runtime=foobar` we should fail immediately and not assume it means "cri-o".

Reply:
Updated now, Thanks for your feedback.